source/features/index.tsx

```diff
@@ -93,6 +93,10 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 		return;
 	}
 
+	if (!document.head.childElementCount) {
```

Comment:
On any of the pages that "were not supposed to be on" like https://github.com/sindresorhus/refined-github/pull/3433/show_toc?base_sha=784cb0c745fba344a24c3bcc9d96092a1a1ce166&sha1=784cb0c745fba344a24c3bcc9d96092a1a1ce166&sha2=ed17f32266803e38aaea74517bdb0345f1388ece#diff-d6ab7dfee8e01c5e0c6d61abd3c7d41c. I get a whole bunch of errors.

Add a comment?

Reply:
That's not a real page, come on